internal/webhookcert/resources.go

```diff
@@ -71,3 +76,16 @@ func updateLogPipelineCRDWithConversionWebhookConfig(ctx context.Context, c clie
 
 	return c.Update(ctx, &logPipelineCRD)
 }
+
+func updateMutatingWebhookConfig(ctx context.Context, c client.Client, caBundle []byte, config Config) error {
+	var mutatingWebhook admissionregistrationv1.MutatingWebhookConfiguration
+	if err := c.Get(ctx, config.MutatingWebhookName, &mutatingWebhook); err != nil {
+		return fmt.Errorf("failed to get mutating webhook: %w", err)
+	}
+
+	for i := range len(mutatingWebhook.Webhooks) {
```

Comment:
I like that you iterated through the webhooks with a loop here to set the CABundle 👍🏻

Can you also adjust the `updateValidatingWebhookConfig()` func to also iterate through the webhooks with a loop instead of explicitly setting the first 2 webhooks. So, in the future if we decided to add more validating webhooks, we won't need to adjust the `updateValidatingWebhookConfig()` func .

Reply:
validating webhook update changed like mutating webhook update